soroban-sdk/test_snapshots/tests/token_client/test_mock_all_auth.1.json

```diff
@@ -128,7 +128,7 @@
             },
             "ext": "v0"
           },
-          15
+          6311999
```

Comment:
Is there some flakiness/non-determinism in terms of how these are generated?

Reply:
I think this is due to the change of using max_entry_ttl as the expiration ledger for the auth entry (https://github.com/stellar/rs-soroban-env/pull/1419). 
```
15 == min_temp_entry_ttl - 1
6311999 == max_entry_ttl - 1
```

Reply:
Ahh, I see, that's due to the env bump, not due to your changes. Thanks for clarifying.